src/Type/Type.php

```diff
@@ -134,7 +134,7 @@ public function setExistingOffsetValueType(Type $offsetType, Type $valueType): T
 
 	public function unsetOffset(Type $offsetType): Type;
 
-	public function getKeysArray(): Type;
+	public function getKeysArray(?Type $filterValueType = null, bool $strict = false): Type;
```

Comment:
1) This a BC break
2) I don't like optional parameters at all.

The solution is to add another method that you'll use in the places where you're passing in the parameters now.

Reply:
fixed